### PR TITLE
Remove leftover openshift flag

### PR DESCRIPTION
**Pull request by vegichan**
- Remove leftover openshift flag
- also remove leftover README document

### CODE REVIEW

**Comment by acabrele** [comment]
Removing the values entries in all locations too?

**Comment by vegichan** [comment]
relates to #66

**Review by sivanov-nuodb — APPROVED** [review]
Thanks!

File: stable/database/README-HOSTPATH.md [hunk 1]
```diff
@@ -1,91 +0,0 @@
-# Using hostPath Storage with DaemonSet
```
Comment (adriansuarez) [inline stable/database/README-HOSTPATH.md:1]: Is hostpath for DaemonSets documented elsewhere?

**Review by adriansuarez — APPROVED** [review]

**Pull request merged**